Kennyj42 sfdc 1 (#1750)
cloudflare/cloudflare-docs: products/cloudflare-one/src/content/tutorials/salesforce-saas.md

   ![Salefsorce configuration](../static/zero-trust-security/access/salesforce.png)

> 1. Navigate to **Security Controls > Single Sign-On Settings**

Reviewer: ```suggestion
1. Navigate to **Security Controls > Single Sign-On Settings**.
```

  * **API name:** (this will pre-populate)
  * **Issuer:** `<your-team-name>.cloudflareaccess.com`, where your-team-name is your [team name](/glossary#team-name).
  * **Identity Provider Certificate**: upload the `.crt` certificate file you’ve created in the previous step. 
  * **EntityID**: `https://[YOUR_SFDC_DOMAIN].my.salesforce.com`
  * **SAML Identity type:** Assertion contains the Federation ID from the User object
  * **Identity Provider Login URL**: This is the SSO endpoint provided in the Teams dashboard for that application.

1. Click **Save**.

  ![Salefsorce configuration](../static/zero-trust-security/access/salesforce-sso.png)
  

> 1. From the navigation panel on the left, click on Domain Management > My Domain. Select your domain.

Reviewer: ```suggestion
1. From the navigation panel on the left, click **Domain Management** > **My Domain** and select your domain.
```

> 1. Scroll to the bottom to find Authentication Configuration. Click Edit and select your Authentication Service you created.

Reviewer: ```suggestion
1. At the bottom, find **Authentication Configuration**. Click **Edit** and select your Authentication Service you created.
```

> 1. (Optional) Select Security Controls > Signle Sign-On Settings > Edit. Select `Disable login with Salesforce credentials` this will enforce all users to sign in through Cloudflare Access.

Reviewer: ```suggestion
1. (Optional) To force all users to sign in through Cloudflare Access:
    1. Click **Security Controls** > **Single Sign-On Settings** > **Edit**. 
    1. Click `Disable login with Salesforce credentials`.
```
